components/dom_crawler.rst

```diff
@@ -219,7 +219,7 @@ Extract attribute and/or node values from the list of nodes::
 
 .. note::
 
-    Special attribute ``_text`` represents a node value.
+    Special attribute ``_text`` represents a node value and `_name`. for a element name.
```

Comment:
The example before shows the actual use case of `_text`:
```
  $attributes = $crawler
         ->filterXpath('//body/p')
         ->extract(array('_text', 'class'))
    ;
```

Could you add/update the example with `_name`? Also use double backticks around name.

Wording looks kinda bad to me, what about: 

```
Special attribute ``_text`` represents a node value, while ``_name`` represents element name.
```
```

Reply:
Okay, I'll fix it now. Translated into English Google.